[2019-05-01 11:25:10,450] {__init__.py:1139} INFO - Dependencies all met for <TaskInstance: Salted_Graph_Practice1.Sense_Copied_Data 2019-05-01T15:24:30.215198+00:00 [queued]>
[2019-05-01 11:25:10,453] {__init__.py:1139} INFO - Dependencies all met for <TaskInstance: Salted_Graph_Practice1.Sense_Copied_Data 2019-05-01T15:24:30.215198+00:00 [queued]>
[2019-05-01 11:25:10,453] {__init__.py:1353} INFO - 
--------------------------------------------------------------------------------
[2019-05-01 11:25:10,453] {__init__.py:1354} INFO - Starting attempt 1 of 1
[2019-05-01 11:25:10,453] {__init__.py:1355} INFO - 
--------------------------------------------------------------------------------
[2019-05-01 11:25:10,462] {__init__.py:1374} INFO - Executing <Task(FileSensor): Sense_Copied_Data> on 2019-05-01T15:24:30.215198+00:00
[2019-05-01 11:25:10,462] {base_task_runner.py:119} INFO - Running: ['airflow', 'run', 'Salted_Graph_Practice1', 'Sense_Copied_Data', '2019-05-01T15:24:30.215198+00:00', '--job_id', '287', '--raw', '-sd', 'DAGS_FOLDER/salted_graph.py', '--cfg_path', '/tmp/tmpa0n6d8ye']
[2019-05-01 11:25:10,736] {base_task_runner.py:101} INFO - Job 287: Subtask Sense_Copied_Data /home/mjcarleb/anaconda3/envs/airflow4/lib/python3.7/site-packages/airflow/configuration.py:214: FutureWarning: The task_runner setting in [core] has the old default value of 'BashTaskRunner'. This value has been changed to 'StandardTaskRunner' in the running config, but please update your config before Apache Airflow 2.0.
[2019-05-01 11:25:10,736] {base_task_runner.py:101} INFO - Job 287: Subtask Sense_Copied_Data   FutureWarning,
[2019-05-01 11:25:10,890] {base_task_runner.py:101} INFO - Job 287: Subtask Sense_Copied_Data [2019-05-01 11:25:10,889] {__init__.py:51} INFO - Using executor SequentialExecutor
[2019-05-01 11:25:11,031] {base_task_runner.py:101} INFO - Job 287: Subtask Sense_Copied_Data [2019-05-01 11:25:11,031] {__init__.py:305} INFO - Filling up the DagBag from /home/mjcarleb/HESWork/CSCI-E29/2019sp-airflow_project-mjcarleb/dags/salted_graph.py
[2019-05-01 11:25:11,052] {base_task_runner.py:101} INFO - Job 287: Subtask Sense_Copied_Data [2019-05-01 11:25:11,051] {cli.py:517} INFO - Running <TaskInstance: Salted_Graph_Practice1.Sense_Copied_Data 2019-05-01T15:24:30.215198+00:00 [running]> on host localhost.localdomain
[2019-05-01 11:25:11,059] {file_sensor.py:60} INFO - Poking for file /home/mjcarleb/HESWork/CSCI-E29/2019sp-airflow_project-mjcarleb/copied/yelp_subset_1.csv
[2019-05-01 11:25:21,073] {file_sensor.py:60} INFO - Poking for file /home/mjcarleb/HESWork/CSCI-E29/2019sp-airflow_project-mjcarleb/copied/yelp_subset_1.csv
[2019-05-01 11:25:31,081] {file_sensor.py:60} INFO - Poking for file /home/mjcarleb/HESWork/CSCI-E29/2019sp-airflow_project-mjcarleb/copied/yelp_subset_1.csv
[2019-05-01 11:25:31,084] {__init__.py:1580} ERROR - Snap. Time is OUT.
Traceback (most recent call last):
  File "/home/mjcarleb/anaconda3/envs/airflow4/lib/python3.7/site-packages/airflow/models/__init__.py", line 1441, in _run_raw_task
    result = task_copy.execute(context=context)
  File "/home/mjcarleb/anaconda3/envs/airflow4/lib/python3.7/site-packages/airflow/sensors/base_sensor_operator.py", line 117, in execute
    raise AirflowSensorTimeout('Snap. Time is OUT.')
airflow.exceptions.AirflowSensorTimeout: Snap. Time is OUT.
[2019-05-01 11:25:31,085] {__init__.py:1611} INFO - Marking task as FAILED.
[2019-05-01 11:25:31,103] {base_task_runner.py:101} INFO - Job 287: Subtask Sense_Copied_Data Traceback (most recent call last):
[2019-05-01 11:25:31,103] {base_task_runner.py:101} INFO - Job 287: Subtask Sense_Copied_Data   File "/home/mjcarleb/anaconda3/envs/airflow4/bin/airflow", line 32, in <module>
[2019-05-01 11:25:31,103] {base_task_runner.py:101} INFO - Job 287: Subtask Sense_Copied_Data     args.func(args)
[2019-05-01 11:25:31,103] {base_task_runner.py:101} INFO - Job 287: Subtask Sense_Copied_Data   File "/home/mjcarleb/anaconda3/envs/airflow4/lib/python3.7/site-packages/airflow/utils/cli.py", line 74, in wrapper
[2019-05-01 11:25:31,103] {base_task_runner.py:101} INFO - Job 287: Subtask Sense_Copied_Data     return f(*args, **kwargs)
[2019-05-01 11:25:31,103] {base_task_runner.py:101} INFO - Job 287: Subtask Sense_Copied_Data   File "/home/mjcarleb/anaconda3/envs/airflow4/lib/python3.7/site-packages/airflow/bin/cli.py", line 523, in run
[2019-05-01 11:25:31,103] {base_task_runner.py:101} INFO - Job 287: Subtask Sense_Copied_Data     _run(args, dag, ti)
[2019-05-01 11:25:31,103] {base_task_runner.py:101} INFO - Job 287: Subtask Sense_Copied_Data   File "/home/mjcarleb/anaconda3/envs/airflow4/lib/python3.7/site-packages/airflow/bin/cli.py", line 442, in _run
[2019-05-01 11:25:31,103] {base_task_runner.py:101} INFO - Job 287: Subtask Sense_Copied_Data     pool=args.pool,
[2019-05-01 11:25:31,103] {base_task_runner.py:101} INFO - Job 287: Subtask Sense_Copied_Data   File "/home/mjcarleb/anaconda3/envs/airflow4/lib/python3.7/site-packages/airflow/utils/db.py", line 73, in wrapper
[2019-05-01 11:25:31,103] {base_task_runner.py:101} INFO - Job 287: Subtask Sense_Copied_Data     return func(*args, **kwargs)
[2019-05-01 11:25:31,103] {base_task_runner.py:101} INFO - Job 287: Subtask Sense_Copied_Data   File "/home/mjcarleb/anaconda3/envs/airflow4/lib/python3.7/site-packages/airflow/models/__init__.py", line 1441, in _run_raw_task
[2019-05-01 11:25:31,103] {base_task_runner.py:101} INFO - Job 287: Subtask Sense_Copied_Data     result = task_copy.execute(context=context)
[2019-05-01 11:25:31,103] {base_task_runner.py:101} INFO - Job 287: Subtask Sense_Copied_Data   File "/home/mjcarleb/anaconda3/envs/airflow4/lib/python3.7/site-packages/airflow/sensors/base_sensor_operator.py", line 117, in execute
[2019-05-01 11:25:31,103] {base_task_runner.py:101} INFO - Job 287: Subtask Sense_Copied_Data     raise AirflowSensorTimeout('Snap. Time is OUT.')
[2019-05-01 11:25:31,103] {base_task_runner.py:101} INFO - Job 287: Subtask Sense_Copied_Data airflow.exceptions.AirflowSensorTimeout: Snap. Time is OUT.
[2019-05-01 11:25:35,496] {logging_mixin.py:95} INFO - [2019-05-01 11:25:35,496] {jobs.py:2562} INFO - Task exited with return code 1
